observation_portal/common/mixins.py

```diff
@@ -12,6 +13,12 @@ def list(self, request, *args, **kwargs):
         return self.get_paginated_response(json_models)
 
 
+class DetailAsDictMixin:
```

Comment:
Mirrors the `ListAsDictMixin` for the detail route. I am using this together with the 
 `ListAsDictMixin` for some of the viewsets in the proposals app because I wanted the serialized response for an object to be the same both in a detail and in a list response.